### PR TITLE
Support multiple columns of locations

### PR DESCRIPTION
**Pull request by brandones**
Screenshot from 2020-03-20 11-21-14

### CODE REVIEW

**Comment by jonathandick** [comment]
Can you add a filter?

File: src/styles.css [hunk 1]
```diff
@@ -87,9 +87,10 @@
   margin-top: 0.625rem;
```
Comment (jonathandick) [inline src/styles.css]: My recent learning has been that margins should probably be in pixels. REM is device dependent (literally the size of an "m"). So margins will be adjusted accordingly and that may not be desired. But will defer to your judgment

**Comment by mseaton** [comment]
Looks great overall @brandones - a few quck comments:

1. Can you confirm that 20 rows will fit without scrolling on a standard display that is used in our clinics?  We want to avoid scrolling if at all possible.

2. Can we make the locations not auto-wrap?  Maybe that location name that wraps is just too long, but if we can configure it to not wrap at all, that would be preferable.

File: src/styles.css (hunk 1 above)
Comment (brandones) [reply src/styles.css]: Last I heard we were an all-rem-all-the-time shop. Also, I didn't add that. This is a larger discussion that we should  have on Slack once we have designer help. Until then I think we should just keep doing what we've been doing.

**Comment by brandones** [comment]
@jonathandick Yeah that'll be a separate PR. Thanks for the review.

**Comment by brandones** [comment]
@mseaton 
1.How tall are the standard displays at our clinics? (I'd tag Dave but is he not on GitHub?)
2. What would you want to happen, instead of wrapping?

**Comment by mseaton** [comment]
> @mseaton
> 1.How tall are the standard displays at our clinics? (I'd tag Dave but is he not on GitHub?)

Asked on chat, I'm not sure.  Does it look good on your laptop with no scrolling?

> 2. What would you want to happen, instead of wrapping?

I think just expanding left/right until some maximum threshold is hit.

**Comment by brandones** [comment]
Yeah, the screenshot I posted is on my laptop screen. Here's a comparison with the existing UI, at the same zoom:

Screenshot from 2020-03-20 12-14-15

**Comment by brandones** [comment]
> I think just expanding left/right until some maximum threshold is hit.

You mean that you would expect that whole first column to widen to fit its largest element (up to a threshold)? Or the text should cut off? Or the text should overflow? Or it should take up the cell horizontally next to it (probably not possible without JS)?

**Comment by brandones** [comment]
Screenshot from 2020-03-23 10-56-11

Tweaked widths just slightly so that all the text fits. Hopefully future implementations will choose location names that are not so long. @mseaton , waiting on approval from you.

**Pull request merged**